### ISSUE TITLE
in layman's terms, what does this do and why would someone use it?

### ISSUE THREAD

**Issue by Dr0nen** [dtinth/sheet.spacet.me#1]
in layman's terms, what does this do and why would someone use it?

**Comment by dtinth** [comment]
This app turns Google Sheets into a public JSON endpoint, which allows anyone to access the data in the sheet in a format that can be easily read by computers. This is useful for making data in Google Sheets available to other applications or websites. Potential use cases include making data in a spreadsheet available to a website or application, or allowing other applications to access data in a spreadsheet.

**Comment by Dr0nen** [comment]
I have no idea how to message you @dtinth , please message taldrone on 5kype
